Rubberduck.Core/UI/CodeExplorer/CodeExplorerControl.xaml

```diff
@@ -5,9 +5,9 @@
              xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
              xmlns:i="http://schemas.microsoft.com/expression/2010/interactivity"
              xmlns:codeExplorer="clr-namespace:Rubberduck.Navigation.CodeExplorer"
-             xmlns:controls="clr-namespace:Rubberduck.UI.Controls"
-             xmlns:converters="clr-namespace:Rubberduck.UI.Converters"
-             xmlns:local="clr-namespace:Rubberduck.UI.CodeExplorer"
+             xmlns:controls="clr-namespace:Rubberduck.UI.Controls;assembly="
+             xmlns:converters="clr-namespace:Rubberduck.UI.Converters;assembly="
+             xmlns:local="clr-namespace:Rubberduck.UI.CodeExplorer;assembly="
```

Comment:
Confused. This works??

Reply:
Compare [The xaml namespaces and namespace mapping docs](https://docs.microsoft.com/en-us/dotnet/framework/wpf/advanced/xaml-namespaces-and-namespace-mapping-for-wpf-xaml), section "Mapping to Current Assemblies": 

> `assembly` can be omitted if the `clr-namespace` referenced is being defined within the same assembly as the application code that is referencing the custom classes. Or, an equivalent syntax for this case is to specify `assembly=`, with no string token following the equals sign.